### PR TITLE
Rename ramonimbao folder to rmi_kb

### PR DESCRIPTION
**Pull request by ramonimbao**
## Description

I would like to stop using my personal name as the folder for my keyboards, and instead use my brand name.

I've renamed the `keyboards/ramonimbao` folder to `keyboards/rmi_kb` and have updated the `info.json` files to reflect the new 'manufacturer' name.

## Types of Changes

- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [x] Keyboard (addition or update)
- [ ] Keymap/layout/userspace (addition or update)
- [ ] Documentation

## Issues Fixed or Closed by This PR

* 

## Checklist

- [x] My code follows the code style of this project: [**C**](https://docs.qmk.fm/#/coding_conventions_c), [**Python**](https://docs.qmk.fm/#/coding_conventions_python)
- [x] I have read the [**PR Checklist** document](https://docs.qmk.fm/#/pr_checklist) and have made the appropriate changes.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [ ] I have added tests to cover my changes.
- [x] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

**Comment by drashna** [comment]
What about the readme's?

**Comment by ramonimbao** [comment]
Right, sorry. Thanks! I forgot about that. I've just updated them now, including a few `rules.mk` files.

**Comment by zvecr** [comment]
Unfortunately there are merge conflicts.

Could you also update `data/mappings/keyboard_aliases.hjson` to reflect the relocated keyboards.

**Comment by zvecr** [comment]
> Done! There are still merge conflicts though. And those files are just ones I did git mv to the new rmi_kb directory.

You still need to resolve them, as its unmergable in the current state.

image

**Comment by ramonimbao** [comment]
Okay, no more merge conflicts as of now.

File: keyboards/rmi_kb/wete/v1/keymaps/via/config.h [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
```
Comment (zvecr) [inline keyboards/rmi_kb/wete/v1/keymaps/via/config.h:16]:
```suggestion
#pragma once 

```

File: keyboards/rmi_kb/tkl_ff/info.json [hunk 2]
```diff
@@ -1,20 +1,12 @@
...
     "keyboard_name": "TKL FF",
-    "manufacturer": "Ramon Imbao",
+    "manufacturer": "RMI-KB",
     "url": "",
-    "maintainer": "Ramon Imbao",
+    "maintainer": "ramonimbao",
     "usb": {
         "vid": "0xB16B",
         "device_version": "0.0.1"
     },
```
Comment (zvecr) [inline keyboards/rmi_kb/tkl_ff/info.json:9]: Not sure why this was removed
```suggestion
    },
    "layout_aliases": {
        "LAYOUT_ansi": "LAYOUT_tkl_f13_ansi",
        "LAYOUT_iso": "LAYOUT_tkl_f13_iso"
    },
    "community_layouts": [
        "tkl_f13_ansi",
        "tkl_f13_iso"
    ],
```

File: keyboards/rmi_kb/squishy65/keymaps/via/config.h [hunk 3]
```diff
@@ -0,0 +1,17 @@
...
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
```
Comment (zvecr) [inline keyboards/rmi_kb/squishy65/keymaps/via/config.h:16]:
```suggestion
#pragma once

```

**Review by zvecr — CHANGES_REQUESTED** [review]

**Review by zvecr — APPROVED** [review]

**Review by waffle87 — APPROVED** [review]

**Pull request merged**